cypress/integration/rich-text/RichTextEditor.spec.ts

```diff
@@ -360,6 +360,73 @@ describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
             richText.expectValue(expectedQuoteValue);
           });
         }
+
+        it('should be deleted if empty when pressing delete', () => {
+          richText.editor.click(); // to set an initial editor.location
+
+          clickDropdownItem(type);
+
+          richText.editor.type('x{enter}');
+
+          cy.findByTestId('toolbar-entity-dropdown-toggle').click();
+          cy.findByTestId('toolbar-toggle-embedded-entry-block').click();
+
+          // To make sure paragraph/heading is present
+          richText.expectValue(
+            doc(
+              block(type, {}, text('x')),
+              block(BLOCKS.EMBEDDED_ENTRY, {
+                target: {
+                  sys: {
+                    id: 'example-entity-id',
+                    type: 'Link',
+                    linkType: 'Entry',
+                  },
+                },
+              }),
+              block(BLOCKS.PARAGRAPH, {}, text(''))
+            )
+          );
+
+          richText.editor
+            .click('bottom')
+            .type('{uparrow}')
+            .type('{uparrow}')
+            .type('{uparrow}')
+            .type('{del}')
+            .type('{del}');
```

Comment:
Need to do this instead of `.type('{uparrow}{uparrow}{uparrow}{del}{del}')` because I think Cypress triggers it too fast and the editor doesn't get it, resulting in flakiness.

Reply:
Could you try it as one command with a `{ delay:` option instead to make it a bit more explicit? (a comment would also help)